parts/k8s/cloud-init/artifacts/cse_install.sh

```diff
@@ -175,12 +188,16 @@ installImg() {
 extractHyperkube() {
   CLI_TOOL=$1
   path="/home/hyperkube-downloads/${KUBERNETES_VERSION}"
+  targetpath="/usr/local/bin"
+  if [[ $OS == $FLATCAR_OS_NAME ]]; then
+    targetpath="/opt"
```

Comment:
sorry @invidian , earlier comment was for kublet only, same question for kubectl here, should we install in /opt/bin?

Reply:
Yeah, it should be OK to install it in `/opt/bin`. I used `/opt`, as it was used for some other binaries IIRC.